README.md

```diff
@@ -31,10 +41,14 @@ You can then use `yarn` to install the project dependencies:
 
 ## Starting the dev server
 
+  To get this app starting the dev server type the following command
+
     $ yarn start
 
 ### Building
 
+  To build this app, simply type the below command:
```

Comment:
Why do you use following up there, below here and nothing down there?

     To build this app, type the following command: